tests/unit/Rules/CnpjTest.php

```diff
@@ -41,38 +39,33 @@ public function testFormattedCnpjsShouldValidate($input)
     public function testUnformattedCnpjsShouldValidates($input)
     {
         $this->assertTrue($this->cnpjValidator->validate($input));
-        $this->assertTrue($this->cnpjValidator->check($input));
-        $this->assertTrue($this->cnpjValidator->assert($input));
     }
 
     /**
      * @dataProvider providerInvalidFormattedCnpj
      * @expectedException Respect\Validation\Exceptions\CnpjException
      */
-    public function testInvalidCnpjShouldThrowCnpjExceptionAndReturnFalseWhenFormatted($input)
+    public function testInvalidCnpjShouldThrowCnpjExceptionWhenFormatted($input)
     {
-        $this->assertFalse($this->cnpjValidator->validate($input));
-        $this->assertFalse($this->cnpjValidator->assert($input));
+        $this->cnpjValidator->assert($input);
     }
 
     /**
      * @dataProvider providerInvalidUnformattedCnpj
      * @expectedException Respect\Validation\Exceptions\CnpjException
      */
-    public function testInvalidCnpjShouldThrowCnpjExceptionAndReturnFalseWhenNotFormatted($input)
+    public function testInvalidCnpjShouldThrowCnpjExceptionWhenNotFormatted($input)
     {
-        $this->assertFalse($this->cnpjValidator->validate($input));
-        $this->assertFalse($this->cnpjValidator->assert($input));
+        $this->cnpjValidator->assert($input);
     }
 
     /**
      * @dataProvider providerInvalidFormattedAndUnformattedCnpjLength
      * @expectedException Respect\Validation\Exceptions\CnpjException
      */
-    public function testCnpjsWithIncorrectLengthShouldThrowCnpjExceptionAndReturnFalse($input)
+    public function testCnpjsWithIncorrectLengthShouldThrowCnpjException($input)
     {
-        $this->assertFalse($this->cnpjValidator->validate($input));
-        $this->assertFalse($this->cnpjValidator->assert($input));
+        $this->cnpjValidator->assert($input);
```

Comment:
Thanks the kind of test we're trying to avoid, instead you should validate just the `validate()` method, since it's the [only existing method on Cnpj rule](https://github.com/Respect/Validation/blob/master/library/Rules/Cnpj.php#L16-L39).